std_msgs/Header header
string object_name
float32 confidence
int32[] bbox_xywh
string message